ci-droid-gui/src/app/routes/form/form.component.spec.ts

```diff
@@ -417,5 +432,48 @@ describe('FormComponent', () => {
       ]);
       expect(component.resources.length).toEqual(1);
     });
+
+    it('should create the request for bulk update', () => {
+      initializeForm();
+      expect(component.createUpdateRequest().email).toBe('dileep.jami@gmail.com');
+    });
+
+    it('should create the request for bulk update and the action is a success', () => {
+      spyOn(component, 'performBulkUpdate').and.callThrough();
+      const sendBulkUpdateActionSpy = spyOn(ciDroidService, 'sendBulkUpdateAction');
+      sendBulkUpdateActionSpy.and.returnValue(of(true));
+      component.resources.length = 5;
+      initializeForm();
+      fixture.nativeElement.querySelector('form').dispatchEvent(new Event('submit'));
+      fixture.detectChanges();
+      expect(component.performBulkUpdate).toHaveBeenCalled();
+      expect(component.email.value).toBe('dileep.jami@gmail.com');
+      expect(ciDroidService.sendBulkUpdateAction).toHaveBeenCalled();
```

Comment:
use [`toHaveBeenCalledWith()`](https://jasmine.github.io/2.0/introduction#section-Spies) instead of `toHaveBeenCalled()`. If you do so you wouldn't need `expect(component.email.value).toBe('dileep.jami@gmail.com');`